### ISSUE TITLE
Will you update to new Androidx watchface API?

### ISSUE THREAD

**Issue by mattwakeman** [qbalsdon/wearOS#6]

**Comment by qbalsdon** [comment]
Hey there! Thanks for the interest in this project. 

I'll give it a go - it looks like they've either taken some serious inspiration from my project or we're just very creepily thinking along the same lines. So I need to rename a few things and move stuff around, but it looks like more a war of names than anything else. 

Problem is this is still alpha so by doing this I become a guinea pig for them. Thanks for the heads up though

**Comment by qbalsdon** [comment]
Hi @mattwakeman - I have had a look around the new API's and the resulting code is a such a significant change from this repository that I think it's best to deprecate this repo as an archive. They seem to have incorporated a lot of my ideas, such as simplifying the renderer to allow for hardware and software, and the removal of the entire engine inner class requirement. 

The structure is very cool (I know, because I think I spearheaded it, or at least "great minds think alike" -eded it :) I found a great article here: https://android-developers.googleblog.com/2021/12/develop-watch-faces-with-stable-jetpack.html

In the meantime I think a lot of what this repo achieves can also be achieved using Samsung's Watch Face Studio: https://developer.android.com/training/wearables/wfs

So in short, thank you for pointing out th new libraries, I think they change the game a lot. I am starting a new position in a few months and will more than likely be doing this in my new role. So you can expect an update with the next 6 months or so along this type or architecture, if the emulator is not up to the task.
